### PR TITLE
[HOTFIX] Fix pinnings for dask.

### PR DESCRIPTION
**Pull request by vyasr**
The pip package pinnings were missed in #1062.

### CODE REVIEW

**Review by benfred — APPROVED** [review]
looks good - thanks for updating!

(btw, I have a PR https://github.com/rapidsai/raft/pull/1083 to update the codespell settings to ignore the changelog because of the initial failure here)

**Review by galipremsagar — APPROVED** [review]

File: python/raft-dask/setup.py
Comment (galipremsagar) [inline python/raft-dask/setup.py:29]: We might want to add this to release version upgrade script at https://github.com/rapidsai/raft/blob/bd2fa8ade237613c769aaf26489a7bf49b353d8c/ci/release/update-version.sh

Though this can be done in 23.02, should be good to merge this PR once @cjnolet approves.

**Review by galipremsagar — APPROVED** [review]

**Review by cjnolet — APPROVED** [review]
LGTM

**Pull request merged**
